ElectronClient/plugins/GotoAnything.jsx

```diff
@@ -260,20 +260,13 @@ class Dialog extends React.PureComponent {
 				}
 			}
 
-			let selectedItemId = null;
-			const itemIndex = this.selectedItemIndex(results, this.state.selectedItemId);
-			if (itemIndex > 0) {
-				selectedItemId = this.state.selectedItemId;
-			} else if (results.length > 0) {
-				selectedItemId = results[0].id;
-			}
-
+			this.itemListRef.current.makeItemIndexVisible(0);
 			this.setState({
-				listType: listType,
-				results: results,
+				listType,
+				results,
+				resultsInBody,
+				selectedItemId: results.length === 0 ? null : results[0].id,
 				keywords: this.keywords(searchQuery),
-				selectedItemId: selectedItemId,
-				resultsInBody: resultsInBody,
```

Comment:
I feel there's too many changes, and some of them aren't necessary. I feel the previous logic also made more sense.

Reply:
ok, I've converted some changes.